docs/UseSigma.md

```diff
@@ -110,3 +111,59 @@ You should NOT import the following rules
 ../timesketch/data/sigma/rules/linux/reverse_shell.yaml
 ../timesketch/data/sigma/rules/linux/recon_commands.yaml
 ```
+
+## Toubleshooting
+
+### How to find issues
+
+#### Logs
+
+In the celery logs, while running the sigma analyzer, you will see something like that:
+
+```shell
+result: Applied 0 tags
+* win_apt_carbonpaper_turla.yml: 0
+...
+* win_syskey_registry_access.yml: 0
+Problematic rules:
+XXXX
+```
+
+The XXX here is the "problem" and you should note those rules. Once you note and identified those rules, it is recommended to take the id and attempt a API call like the following:
+
+```python
+from timesketch_api_client import config
+ts = config.get_client()
+rule = ts.get_sigma_rule("c0478ead-5336-46c2-bd5e-b4c84bc3a36e")
+print(rule.es_query)
+```
+
+Where the ID is the id of your problematic rule. This will hopefully give you more insight from the web server logs of what caused the problem. E.g.
+"Aggregations not implemented for this backend"
+It is then recommended to move those rules to a separate folder, maybe even creating a small shell script that does that for you once you pull upstream rules from the Sigma repository.
+
+### How to verify issues
+
+#### Timesketch API / logs
+
+If you have doubt if a rule does work, take the uuid and run python code mentioned above.
+
+#### sigmac
+
+Another option is to run the rule against the official sigma client with the Timesketch sigma mapping file.
+
+For our example from above:
+
+```shell
+sigma/tools/sigma$ python3 sigmac.py -t es-qs --config ../../../sigma_config.yaml ../../rules/windows/image_load/sysmon_mimikatz_inmemory_detection.yml
+An unsupported feature is required for this Sigma rule (../../rules/windows/image_load/sysmon_mimikatz_inmemory_detection.yml): Aggregations not implemented for this backend
+Feel free to contribute for fun and fame, this is open source :) -> https://github.com/Neo23x0/sigma
+```
+
+### What to do with problematic rules
+
+To reduce load on the system it is recommended to not keep the rules in the directory, as it will cause the exception every time the rules folders are parsed (a lot!).
```

Comment:
```suggestion
To reduce load on the system it is recommended to not keep the problematic rules in the directory, as it will cause the exception every time the rules folders are parsed (a lot!).
```